airbyte-tests/src/acceptanceTests/java/io/airbyte/test/acceptance/BasicAcceptanceTests.java

```diff
@@ -755,37 +516,22 @@ public void testIncrementalDedupeSync() throws Exception {
     assertNormalizedDestinationContains(expectedNormalizedRecords);
   }
 
-  @RetryingTest(3)
-  @Order(14)
-  public void testCheckpointing() throws Exception {
-    final SourceDefinitionRead sourceDefinition = createE2eSourceDefinition();
-    final DestinationDefinitionRead destinationDefinition = createE2eDestinationDefinition();
-
-    final SourceRead source = createSource(
-        "E2E Test Source -" + UUID.randomUUID(),
-        workspaceId,
-        sourceDefinition.getSourceDefinitionId(),
-        Jsons.jsonNode(ImmutableMap.builder()
-            .put("type", "EXCEPTION_AFTER_N")
-            .put("throw_after_n_records", 100)
-            .build()));
-
-    final DestinationRead destination = createDestination(
-        "E2E Test Destination -" + UUID.randomUUID(),
-        workspaceId,
-        destinationDefinition.getDestinationDefinitionId(),
-        Jsons.jsonNode(ImmutableMap.of("type", "SILENT")));
-
+  @Test
+  @Order(12)
+  public void testIncrementalSync() throws Exception {
+    LOGGER.info("Starting testIncrementalSync()");
```

Comment:
Just for my own curiosity and understanding - why did you choose to move the `testCheckpointing()` test to `AdvancedAcceptanceTests` and to keep the `testIncrementalSync()` test in `BasicAcceptanceTests`?